app/containers/AppContainer.js

```diff
@@ -19,17 +15,15 @@ import {
   Button
 } from 'react-native';
 
+const path = 'https://movied.herokuapp.com/popular';
```

Comment:
Moved to a folder used for constants like Path for popular movies, or the path for the poster.